frontend/src/components/form/SubmitButton.js

```diff
@@ -1,11 +1,14 @@
-import { useEffect } from "react";
 import useFormContext from "../../hooks/useFormContext";
 import Button from "../@common/Button/Button";
 
-const SubmitButton = () => {
+const SubmitButton = ({ children }) => {
   const { isValid, isEmpty } = useFormContext();
 
-  return <Button disabled={!isValid || isEmpty}>제출</Button>;
+  return (
+    <Button disabled={!isValid || isEmpty}>
+      {children ? children : "제출"}
```

Comment:
```suggestion
      {children ?? "제출"}
```
로 더 간략하게 표현해볼 수 있을 것 같아요.